.storybook/preview.ts

```diff
@@ -1,5 +1,5 @@
 import { addParameters } from '@storybook/react';
-import '../src/styles/index.css';
+import '../src/styles/index.scss';
```

Comment:
I've switched this to scss for now since `css` tokens are not being inlined in the build, but `scss` tokens are. This is something I'll have to check in the rollup config, but for some reason, its ignoring css.